src/main/java/com/example/search/engine/configuration/Properties.java

```diff
@@ -0,0 +1,15 @@
+package com.example.search.engine.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class Properties {
+    @Value("${illegalSymbols}")
```

Comment:
This is not a typical way of injecting properties. 
Instead of @Value and @Component please use @ConfigurationProperties and @Value and
@ConstructorBinding together with @Configuration and @EnableConfigurationProperties used in Application class or separate Configuration class.